### PR TITLE
uninstalled redundant dependency and fixed a build error

### PR DESCRIPTION
**Pull request by larrycoal**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -17,4 +17,4 @@ FIREBASE_MEASUREMENT_ID=
 SENTRY_DSN=
 
 # Configs
-FIREBASE_ANALYTICS_ENABLED=
+FIREBASE_ANALYTICS_ENABLED=
```
Comment (razbakov) [inline .env.example:20]: please ensure there is always a new line in the end of the file

**Review by razbakov — CHANGES_REQUESTED** [review]

**Comment by larrycoal** [comment]
changes made

**Review by razbakov — APPROVED** [review]

**Comment by razbakov** [comment]
node-sass is required somewhere in build command:

```
10:24:03 AM: $ yarn build
10:24:04 AM: yarn run v1.22.4
10:24:04 AM: $ nuxt build
10:25:43 AM: [fatal] Nuxt build error
10:25:43 AM:   ERROR in ./assets/css/vendors.scss (./node_modules/css-loader/dist/cjs.js??ref--7-oneOf-1-1!./node_modules/postcss-loader/src??ref--7-oneOf-1-2!./node_modules/sass-loader/dist/cjs.js??ref--7-oneOf-1-3!./assets/css/vendors.scss)
10:25:43 AM:   Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
10:25:43 AM:   Error: Cannot find module 'node-sass'
```

Full build logs available here - https://app.netlify.com/sites/wedance/deploys/60fed131994ac80007db911f

**Review by razbakov — CHANGES_REQUESTED** [review]
Please split into 3 pull requests

**Pull request closed without merging**